wordpress-mobile/WordPress-iOS
# ULS Get Started: password managers

Ref: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/issues/404
Auth PR: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/438

This uses the Auth changes to support 1Password and password autofill from the keyboard.

To test:

1Password:
- Enable the `unifiedWordPress` feature flag.
- Note I have disabled the iOS version check, so you can test this on an iOS13 device.
- On a device with 1Password set up, go to the Get Started view.
- Tap on the 1Password button on the email field, and verify you can log in.

Password autofill:
- Tap in the email field to display the keyboard.
- Verify you can now select an account from the keyboard.
- Select one and verify you can log in.

![managers](https://user-images.githubusercontent.com/1816888/92161965-9cfc4800-edee-11ea-959b-9ada7153679f.jpg)

PR submission checklist:
